doc/user_guide/transform.rst

```diff
@@ -656,50 +656,50 @@ labels.
 
 Window Transform
 ~~~~~~~~~~~~~~~~
-The Window transform performs calculations over sorted groups of data objects.
-These calculations including ranking, lead/lag analysis, and aggregates such
-as running sums and averages. Calculated values are written back to the
-input data stream, where they can be referenced in encodings.
-
-For example, consider the following chart showing time spent on various
-activities during a day:
+The window transform performs calculations over sorted groups of data objects.
+These calculations include ranking, lead/lag analysis, and aggregates such as cumulative sums and averages.
+Calculated values are written back to the input data stream, where they can be referenced by encodings.
 
+For example, consider the following cumulative frequency distribution:
 
 .. altair-plot::
-
     import altair as alt
-    import pandas as pd
-
-    activities = pd.DataFrame({'Activity': ['Sleeping', 'Eating', 'TV', 'Work', 'Exercise'],
-                               'Time': [8, 2, 4, 8, 2]})
-
-    alt.Chart(activities).mark_bar().encode(
-        x='Time:Q',
-        y='Activity:N'
-    )
-
-You might wish to plot these bars in units of percentage of total time rather than
-in units of hours. You can do this by combining a calculate transform with a
-window transform, using :meth:`~Chart.transform_window`:
-
-.. altair-plot::
-
-    alt.Chart(activities).transform_window(
-        TotalTime='sum(Time)',
-        frame=[None, None]
-    ).transform_calculate(
-        PercentOfTotal="100 * datum.Time / datum.TotalTime"
-    ).mark_bar().encode(
-        x='PercentOfTotal:Q',
-        y='Activity:N'
-    )
-
-In the window transform, we specify ``frame=[None, None]``, which indicates that
-the aggregation at each point is performed on the entire dataset.
+    from vega_datasets import data
 
-Window transforms are quite flexible, and are not yet well documented within
-Altair. For more information on the arguments of the window transform, see
-:class:`WindowTransform`, or see the
-`Vega-Lite window transform examples <https://vega.github.io/vega-lite/docs/window.html>`_.
+    movies = data.movies()
```

Comment:
For large datasets like movies, we should prefer ``data.movies.url`` to avoid embedding the full dataset into the main HTML page.

Reply:
Seems perfectly reasonable to me!